test/nodes/test_webhook_scheduler.py

```
@@ -0,0 +1,691 @@
+            )
+
+    def test_invalid_cron_too_many_fields(self):
+        with pytest.raises(ValueError, match='5 fields'):
+            ScheduleCreate(
+                pipeline_id='my-pipeline',
+                cron_expression='* * * * * *',
+                name='bad-cron',
+            )
+
+    def test_empty_pipeline_id_rejected(self):
+        with pytest.raises(ValueError):
```

Ruff (0.15.7)

[warning] `pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)